tests/test_graph_synthetic.py

```diff
@@ -63,49 +65,6 @@ def test_sample_categorical(self):
         np.random.seed(1)
         attribute = self.synthetic_graph._categorical_attributes[0]
         self.assertEqual(4, self.synthetic_graph.sample_categorical(attribute))
-    
-    def test_plot_sample_categorical(self):
-        np.random.seed(2)
-        attribute = self.synthetic_graph._categorical_attributes[0]
-        data = []
-        for n in range(0, 2000):
-            data.append(self.synthetic_graph.sample_categorical(attribute))
-        
-        hist, edges = np.histogram(data, bins=[1.0, 1.75, 2.5, 3.25, 4.25, 5.25, 6.25, 7.25, 8], density=False)
-        self.assertEqual(list(hist), [24, 45, 374, 379, 201, 81, 72, 824])
-
-        # plots
-        expected_hist = self.expected_profile["categorical_distribution"]["pop"]["bin_counts"]
-        hist = hist/np.max(hist)
-        expected_hist = expected_hist/np.max(expected_hist)
-
-        num_bin = 8
-        bin_lims = np.linspace(0,1,num_bin+1)
-        bin_centers = 0.5*(bin_lims[:-1]+bin_lims[1:])
-        bin_widths = bin_lims[1:]-bin_lims[:-1]
-
-        fig, (ax1,ax2) = plt.subplots(nrows = 1, ncols = 2)
-        ax1.bar(bin_centers, hist, width = bin_widths, align = 'center')
-        ax2.bar(bin_centers, expected_hist, width = bin_widths, align = 'center', alpha = 0.5)
-        ax1.set_title('amplitude-normalized expected distribution')
-        ax2.set_title('amplitude-normalized computed distribution')
-        plt.show()
-
-    def test_plot_sample_continuous(self):
-        np.random.seed(5)
-        attribute = self.synthetic_graph._continuous_attributes[0]
-        data = self.synthetic_graph.sample_continuous(attribute, 2000)
-        properties = self.expected_profile["continuous_distribution"][attribute]["properties"]
-        distribution_continuous_test = st.norm(loc=properties[0], scale=properties[1])
```

Comment:
should this still exist, but do the same assertion as above?
running the func should still be validated: `data = self.synthetic_graph.sample_continuous(attribute, 2000)`